mockapplication/src/main/java/com/ss/benchmark/MockService.java

```diff
@@ -75,6 +75,9 @@ private void createEchoStubs(){
                       get(urlMatching("/echodelayserv/echo/.*"))
                       .willReturn(aResponse().withStatus(200).withHeader("Content-Type", "application/json").withBody(Payloads.SHORT_JSON)));
 
+        wireMockServer.stubFor(
```

Comment:
Move this stub above wildcard stubbing. Otherwise,  wiremock would match wildcard one for echo/long URL.